### PR TITLE
Add missing where clause for effective from and to dates

### PR DESCRIPTION
**Pull request by caoimheb**

### CODE REVIEW

File: seed/views/00901_team_case_overview.js [hunk 1]
```diff
@@ -18,9 +18,12 @@ exports.seed = function (knex, promise) {
...
     JOIN app.offender_manager om ON om.id = wo.offender_manager_id
     JOIN app.team t ON wo.team_id = t.id
-    LEFT JOIN app.workload_points_calculations wpc ON wpc.workload_id = w.id
+    JOIN app.workload_points_calculations wpc ON wpc.workload_id = w.id
     JOIN app.workload_points wp ON wpc.workload_points_id = wp.id 
-    GROUP BY wo.id;`
+    JOIN app.workload_report wr ON wr.id = wpc.workload_report_id
+  WHERE wr.effective_from IS NOT NULL
+    AND wr.effective_to IS NULL
+  GROUP BY wo.id;`
```
Comment (tsheppard01) [inline seed/views/00901_team_case_overview.js]: Will this script be run during migrations? I thought once a script was run once it was ignored every other time?

Comment (dconey646) [reply seed/views/00901_team_case_overview.js]: So, because it is raw sql, its executed to drop the view first, then execute it each time without fail.

Comment (dconey646) [reply seed/views/00901_team_case_overview.js]: `return knex.schema
    .raw('DROP VIEW IF EXISTS app.ldu_caseload_view;')
    .raw('SET ARITHABORT ON')
    .raw(sql)`

Comment (caoimheb) [reply seed/views/00901_team_case_overview.js]: Yeh, views are run each time regardless, as a seed rather than a migration

**Pull request merged**